src/componentLibrary/badge.css

```diff
@@ -0,0 +1,71 @@
+.avatar-images {
+  border: 1px solid var(--gray-border);
+  padding: 2rem;
+  border-radius: 10px;
+  text-align: center;
+  margin: var(--margin-width);
+  display: flex;
+}
+.avatar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+.avatar-md {
+  width: 4.5rem;
+  height: 4.5rem;
+  border-radius: 50%;
+}
+.avatar-badge {
+  position: relative;
+  border: 2px solid var(--white);
+  border-radius: 50%;
+  padding: 0rem 0.5rem;
+  right: 1rem;
+  bottom: 0.5rem;
+}
+.green-dot {
+  background-color: var(--success);
+}
+.red-dot {
+  background-color: var(--danger);
+}
+
+.button_main {
```

Comment:
You should use consistent naming convention. As you have used  `-` everywhere for className, so you can use that here as well instead of `_`